modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/PlacementDriverManager.java

```diff
@@ -278,6 +278,11 @@ private void recoverInternalComponentsBusy() {
 
     private PlacementDriver createPlacementDriver() {
         return new PlacementDriver() {
+            @Override
+            public boolean isActualAt(HybridTimestamp timestamp) {
+                return metastore.clusterTime().currentSafeTime().compareTo(timestamp) >= 0;
```

Comment:
Why there's no 'ts.addPhysicalTime(clockService.maxClockSkewMillis());'

Internally methods like getPrimaryReplica uses `clockService.after` that compares timestamps within the context of clock skew.
```
    default boolean after(HybridTimestamp firstTimestamp, HybridTimestamp anotherTimestamp) {
        return compareWithClockSkew(firstTimestamp, anotherTimestamp) > 0;
    }
```

Reply:
Implementations of methods like `getAssignments()` do not add max clock skew when calculating the wait time. I suppose that a 'not adding' version is more universal as the caller can add the max clock skew (this is what I did: I add it externally to this method)